cmd/dra-example-kubeletplugin/discovery.go

```diff
@@ -20,36 +20,53 @@ import (
 	"fmt"
 	"math/rand"
 	"os"
+	"strconv"
+	"strings"
 
 	resourceapi "k8s.io/api/resource/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	"k8s.io/utils/ptr"
 
+	semver "github.com/Masterminds/semver/v3"
```

Comment:
I see now that the exact library used for CEL expressions in Kubernetes is https://pkg.go.dev/github.com/blang/semver/v4@v4.0.0. I don't imagine we'd see any practical differences with this one, but it'd be nice not to have to worry about that.